src/amo/actions/featured.js

```diff
@@ -0,0 +1,31 @@
+import {
+  FEATURED_GET,
+  FEATURED_LOADED,
+  FEATURED_FAILED,
+} from 'core/constants';
+
+
+export function getFeatured({ addonType }) {
+  if (!addonType) {
+    throw new Error('addonType must be set');
+  }
+
+  return {
+    type: FEATURED_GET,
+    payload: { addonType },
+  };
+}
+
+export function loadFeatured({ addonType, entities, result }) {
+  return {
+    type: FEATURED_LOADED,
+    payload: { addonType, entities, result },
+  };
+}
+
+export function failFeatured({ addonType }) {
```

Comment:
Everything related to `FEATURED_FAILED` doesn't seem to be used in any meaningful way. I see that the action gets dispatched when the exception fails but that's it. I say we remove it all for now until we hook it up async-connect errors properly. In other words, it will probably be a YAGNI feature once we address https://github.com/mozilla/addons-frontend/issues/1521